src/chkstat.cpp

```diff
@@ -550,11 +546,17 @@ bool Chkstat::parseProfile(const std::string &path)
         // profile entry seen.
         if (line[0] == '+')
         {
-            const auto good = parseExtraProfileLine(line, last_entry);
+            if (last_key.empty())
```

Comment:
I have no strong feelings about this, but we could also check `m_profile_entries.empty() || *m_profile_entries.rbegin().caps.valid()` here and remove the `last_key` variable.

Reply:
I don't think that would work since `m_profile_entries` is a `std::map` and `m_profile_entries.rbegin()` doesn't return the most recent entry.

Reply:
Oh yes, seems I was mixing up insertion order and sort order in my head here.